hooks/event-conversion-utils.ts

```diff
@@ -164,35 +195,49 @@ class EventsAirtableRecords extends AirtableRecords {
       nextPage()
     })
 
-    return Promise.resolve(seminarSeriesEvents)
+    const sortedSeminarSeriesEvents = this.sortEventsByStartDate(seminarSeriesEvents, days > 0 ? 'asc' : 'desc')
+    return Promise.resolve(sortedSeminarSeriesEvents)
   }
 
-  async convertToCommunityEvent (record: any): Promise<CommunityEvent> {
+  /**
+   * Convert an Airtable record to a CommunityEvent.
+   * @param record The Airtable record to convert.
+   * @returns The converted CommunityEvent.
+   */
+  async convertToCommunityEvent (record: Record<string, any>): Promise<CommunityEvent> {
     const event = {
-      title: this.getName(record),
-      types: this.getTypes(record),
-      image: await this.getImage(record),
-      location: this.getLocation(record),
-      regions: this.getRegions(record),
+      endDate: record.get(this.recordFields!.endDate) || '',
+      location: record.get(this.recordFields!.location) || WORLD_REGIONS.tbd,
+      regions: record.get(this.recordFields!.regions) || [WORLD_REGIONS.tbd],
+      title: record.get(this.recordFields!.name) || '',
+      to: record.get(this.recordFields!.website) || '',
+
```

Comment:
non-blocking question: 
is this empty line here for a reason?

Reply:
I was just trying to visually separate the properties that get their data using `record.get()` from the one using other more complex functions. Not really necessary, just thought that it helps when reading.